pkg/descheduler/apis/config/types_pluginargs.go

```diff
@@ -145,3 +149,19 @@ type MigrationControllerArgs struct {
 	// DefaultDeleteOptions defines options when deleting migrated pods and preempted pods through the method specified by EvictionPolicy
 	DefaultDeleteOptions *metav1.DeleteOptions
 }
+
+type MigrationFrozenObjectType string
+
+const (
+	MigrationFrozenWorkload MigrationFrozenObjectType = "workload"
+)
+
+type FrozenGateMap map[MigrationFrozenObjectType]MigrationFrozenGate
+
+// MigrationFrozenGate means that if the specified dimension has multiple migrations within the configured time period
+// and exceeds the configured threshold, it will be frozen for a certain period of time.
+type MigrationFrozenGate struct {
+	Duration       metav1.Duration
+	MaxMigrating   int
```

Comment:
How is the duration refreshed? After the previous duration is over, start a new round of counting?

Reply:
Yes. If there is no new eviction operation within the period specified by the duration, a new round will be started after the next eviction is initiated, and the number of evictions will be recorded; otherwise, the time will be extended after the last eviction operation. Similar to a window mechanism, the time window will be adjusted passively.

Reply:
updated.